patroni/ha.py

```diff
@@ -613,6 +617,8 @@ def _run_cycle(self):
                 if self.sysid_valid(self.cluster.initialize) and self.cluster.initialize != self.state_handler.sysid:
                     logger.fatal("system ID mismatch, node %s belongs to a different cluster: %s != %s",
                                  self.state_handler.name, self.cluster.initialize, self.state_handler.sysid)
+                    logger.info("Removing Data dir, allowing DB to bootstrap")
```

Comment:
This action is really questionable and dangerous. We should not touch data directory, it might be that it is the last good copy of your cluster!